isvcs/container.go

```diff
@@ -352,7 +353,12 @@ func (svc *IService) create() (*docker.Container, error) {
 	// service-specific volumes
 	if svc.Volumes != nil && len(svc.Volumes) > 0 {
 		for src, dest := range svc.Volumes {
-			hostpath := svc.getResourcePath(src)
+			var hostpath string
+                        if src == "servicedLogDir" {
```

Comment:
Since `servicedLogDir` is a kind of key or symbolic, I suggest a global variable (defined at the top of this file), named something like `UseServicedLogDir`, and use that new variable in the Volumes definition in logstash.go as well.